deepspeed/utils/z3_leaf_module.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import torch
+from typing import List, Type
+
+default_z3_leaf_module_classes = []
+
+try:
+    from transformers.models.mixtral.modeling_mixtral import MixtralSparseMoeBlock
+    default_z3_leaf_module_classes.append(MixtralSparseMoeBlock)
+except ImportError:
+    pass
```

Comment:
I don't think it is a good idea to make transformers a deepspeed dependency. I think it is okay to initialize list to empty for now, or to populate with just DeepSpeed-MoE.